src/services/identity/ReposService.ts

```diff
@@ -4,10 +4,9 @@ import { retry } from "@octokit/plugin-retry"
 import { Octokit } from "@octokit/rest"
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { GetResponseTypeFromEndpointMethod } from "@octokit/types"
-import git from "isomorphic-git"
```

Comment:
Can we also delete the dependency from package.json? I think this is the only place that `isomorphic-git` is being used.

Reply:
good point, will do

Reply:
done